providers/base/units/stress/jobs.pxu

```diff
@@ -603,51 +603,77 @@ _description:
  VERIFICATION:
      Verify the system is not frozen and the wifi and bluetooth applets are still visible and functional
 
+unit: template
+template-resource: cpuinfo
+template-filter: cpuinfo.type == 'GenuineIntel'
+template-unit: job
 plugin:shell
 category_id: com.canonical.plainbox::stress
-id: stress/s2idle_pm-graph_30
+id: stress/s2idle_pm-graph_30_{model_number}_{model_version}_{model_revision}
```

Comment:
What's the purpose for using a template for this job?

Reply:
The purpose is to make the job be seen on Intel platform only.

Reply:
Why not using

```
requires:
  cpuinfo.type == 'GenuineIntel'
```

instead? This way, you don't need a template and the job will be skipped on every platform except Intel ones.